### PR TITLE
Dimension of element with "position: static" should not affect viewport visibility

### PR DESCRIPTION
**Pull request by neofreko**
## Proposed changes

Element with `position: static` or undefined position CSS style would have 0 height.
Current `isVisibleWithinViewport` does not take into account of this case yet.
This PR will address this additional case.

Currently, `isVisibleWithinViewport` will return `{}` in the case non non-existent element. This PR will interprete non-array result from `selectorExecute` as `false`.

`err.message.indexOf('NoSuchElement') > -1` currently interpreted into `true` value. This PR rectify this to `false`. There's no test for this due to above issue.

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: lib/commands/isVisibleWithinViewport.js [hunk 1]
```diff
@@ -55,17 +55,15 @@ module.exports = function isVisibleWithinViewport (selector) {
...
 
     return resultPromise.then((res) => {
-        if (Array.isArray(res) && res.length === 1) {
-            return res[0]
-        }
-
-        return res
+        if (Array.isArray(res)) {
+            if (res.length === 1) { return res[0] } else { return res }
+        } else { return false }
```
Comment (christian-bromann) [inline lib/commands/isVisibleWithinViewport.js]: no else needed here

Comment (neofreko) [reply lib/commands/isVisibleWithinViewport.js]: But the function will return `undefined` when res is not array?

Comment (christian-bromann) [reply lib/commands/isVisibleWithinViewport.js]: you still can return `false` just not in the else

**Comment by christian-bromann** [comment]
Something is wrong with the rebase, I can't merge it like that.

**Comment by neofreko** [comment]
Yeah. I'm not sure what happened as well. I'll start over.

**Pull request closed without merging**